MCDataProducts/inc/SimTimeOffset.hh

```diff
@@ -0,0 +1,15 @@
+#ifndef MCDataProducts_SimTimeOffset_hh
+#define MCDataProducts_SimTimeOffset_hh
+//
+//  Simple class to represent a global time offset, used in resampling mixing
+//  Original author: David Brown (LBNL) 4/15/2021
+//
+#include <vector>
+namespace mu2e {
+  struct SimTimeOffset {
+    SimTimeOffset(double toff=0.0) : timeOffset_(toff) {}
+    double timeOffset_; // global event time offset
+  };
+  typedef std::vector<SimTimeOffset> SimTimeOffsetCollection;
```

Comment:
SimTimeOffsetCollection serves no useful purpose but adds confusion.
Please remove this typedef, it is not used anywhere besides its own dictionary generation.

Reply:
Its leftover from when it appeared I would need to create a collection of these to handle the case of N>1 resampling events.  Now we've agreed to hardcode that to 1 I can remove it.